src/qtAliceVision/SequenceCache.hpp

```diff
@@ -75,6 +81,7 @@ class SequenceCache : public QObject, public ImageServer
     Q_SIGNAL void requestHandled();
 
   private:
+    const bool _isAsync = true;
```

Comment:
Does it make sense to have a const variable for this? The async fetching is set using it but it will always be set to true no matter what.

Reply:
Yes, it is for debug purposes and may be upgraded later.